src/client/TargetsServerClient.js

```diff
@@ -14,39 +14,46 @@ class TargetClient {
     });
   }
 
+  static getMyTargets() {
+    return new Promise((resolve, reject) => {
+      instance.get('/targets').then((({ data })=> {
+        resolve(data);
+      })).catch(error => {
+        reject(error.message);
+      });
+    });
+  }
+
   static createTarget(target) {
-    return new Promise((resolve, reject)=> {
-      instance.post('/targets', target).then((res=> {
-        const {data} = res;
+    return new Promise((resolve, reject) => {
```

Comment:
You are right, but here he is returning only the `data` of the response( https://github.com/joaquinTop/targetMVD-web-react/pull/15/files#diff-b352bfe01578c262d1604047c637c44fR29)

With your approach you give access to the entire response. It's better to return just the necessary to the action function.